rest/model/codegen.go

```diff
@@ -63,84 +63,113 @@ type modelConversionInfo struct {
 // ModelMapping maps schema type names to their respective DB model
 type ModelMapping map[string]string
 
-func Codegen(schema string, config ModelMapping) ([]byte, error) {
+func Codegen(schema string, config ModelMapping) ([]byte, []byte, error) {
```

Comment:
This function should have a comment explaining the return types so future programmers don't get confused